robot/Cumulus/tests/browser/levels/create_edit_delete_level.robot

```diff
@@ -8,115 +8,102 @@ Library         cumulusci.robotframework.PageObjects
+    Navigate To And Validate Field Value                Maximum Amount (<)      contains    ${max_amount}
+    Go to edit level page                               ${level_id}
     Enter Level Values
-    ...            Minimum Amount=0.01
-    ...            Maximum Amount=0.99
-    Set Focus To Element   xpath: //input[@value='Save']
-    Click Button  Save
-    #adding a workaround to go back to levels tab due to core issue
-    Current Page Should Be     Details    Level__c
-    Go To Page         Details         Level__c        object_id=${level_id}
-    Navigate To And Validate Field Value   Minimum Amount (>\=)    contains    0.01
-    Navigate To And Validate Field Value   Maximum Amount (<)     contains    0.99
-    Navigate To And Validate Field Value    Source Field    contains    npo02__SmallestAmount__c
-
-3 Validate Level Assignment in Batch Job
-    [tags]  unstable
-    # --------------------------------
-    # Modify the SmallestGift field to allow the level to be applied
-    # --------------------------------
-    &{contact} =  API Create Contact
-    Set Global Variable     ${contact_id}       &{contact}[Id]
-    Go To Page    Details    Contact       object_id=${contact_id}
-    Scroll Element Into View    text:Donation Totals
-    Click Button       title:Edit Smallest Gift
-    Wait For Locator  record.footer
-    Populate Field          Smallest Gift     0.75
-    Click Record Button     Save
+    ...                                                 Minimum Amount=${minamount_to_edit}
+    ...                                                 Maximum Amount=${maxamount_to_edit}
+    Enter Level Dd Values
+    ...                                                 Source Field=Smallest Gift
+
+    Click Button                                        Save
+    Go To Page                                          Details
+    ...                                                 Level__c
+    ...                                                 object_id=${level_id}
+
     Wait Until Loading Is Complete
-    Navigate To And Validate Field Value          Smallest Gift    contains    $0.75    section=Donation Totals
-    # --------------------------------
-    # Open NPSP Settings and run the Levels batch job
-    # --------------------------------
-    Open NPSP Settings      Bulk Data Processes         Level Assignment Batch
-    Click Settings Button    idPanelLvlAssignBatch    Run Batch
-    Wait For Batch To Process        LVL_LevelAssign_BATCH    Completed
-    # --------------------------------
-    # Return to the Contact to validate the updated Level field
-    # --------------------------------
-    Go To Page    Details    Contact       object_id=${contact_id}
-    Navigate To And Validate Field Value          Level    contains    ${level_name}    section=Donation Totals
 
+    Navigate To And Validate Field Value    Maximum Amount (<)     contains       ${maxamount_to_edit}
```

Comment:
u missed the step to validate min amount field value

Reply:
It is there, After creating the customer, I am updating the small amount value using the API call as The key goal of the testcase after talking to people is not if we are able to update the value from UI, but does the level get applied correctly.

Salesforce Update                       Contact                   ${data}[contact][Id]  npo02__SmallestAmount__c=${contact_smallestvalue}

Reply:
yes, Thanks added it